
lidar_config:
  broadcast_code: ""
  enable_connect: true
  enable_fan: true
  return_mode: 0
  coordinate: 0
  imu_rate: 0
  extrinsic_parameter_source: 0
  enable_high_sensitivity: true
